opm/output/eclipse/Summary.cpp

```diff
@@ -2021,13 +2024,52 @@ quantity group_efficiency_factor(const fn_args& args)
         return zero;
     }
 
-    const auto gefac =
-        args.schedule[args.sim_step].groups(args.group_name)
-        .getGroupEfficiencyFactor();
+    const auto sched = args.schedule[args.sim_step];
```

Comment:
Tiny nit: This statement copies the `ScheduleState` object.  Now, those aren't **huge** since they mostly contain `shared_ptr<>`s, but on the other hand making a copy of this on (almost) every call to `group_efficiency_factor()` could add up.  Simple remedy is a one-character change to take a reference instead:
```C++
const auto& sched = args.schedule[args.sim_step];
```

Reply:
Oops... fixed now. Thanks!

Reply:
> Oops... fixed now. Thanks!

Much appreciated!  I'll start updating the reference solutions and then I'll merge this a little later.